tests/test_param_routing.py

```diff
@@ -102,7 +109,7 @@ def test_no_extra_meta(wrapper_class, build_fn):
     # with user kwargs
     clf = wrapper_class(model=build_fn, model__hidden_layer_sizes=(100,))
     clf.fit(X, y)
-    assert set(clf.get_meta().keys()) == wrapper_class._meta
+    assert wrapper_class._meta.issubset(set(clf.get_metadata().keys()))
```

Comment:
:-1:

Reply:
Idem above. Swapping out for hardcoded values.